### ISSUE TITLE
update whitelist with more current microsoft email domains

### ISSUE THREAD

**Issue by jnewman67** [schweikert/postgrey#57]
currently the whitelist only includes 3 domains for Microsoft:
bigfish.com
frontbridge.com
microsoft.com

but that list is from 2010, and they're using more now.  should probably add
outlook.com
live.com

there are probably more I'm not thinking of.

thanks.

**Comment by fleish** [comment]
The current file @ https://github.com/schweikert/postgrey/blob/master/postgrey_whitelist_clients was last updated 6 months ago and already includes outlook.com & outbound.protection.outlook.com FWIW

**Comment by jnewman67** [comment]
Sorry that I did not get to respond back to this in time (I started, but got sidetracked about 4 weeks ago :)

I meant to respond with my reason for my original post:

I had an issue with emails from live.com taking over 24 hours to get through.  After tracking down the history of one email that was trying to come through, I figured out why it was taking so long.  Live.com (and I assume other MS domains) uses multiple IP addresses to send out emails - even emails that are requeued after being bounced by postgrey the first time.  In all, it took live.com over 15 unique IP addresses before it reused one again, and the email got through.  Adding live.com to the whitelist solved the problem and kept emails flowing nicely.

To date (I just checked), live.com isnt' in the whitelist.  I'm not up-to-date on all the domains MS uses for sending out email, but I'm sure there are others that might not be on the list - hence the reason for my post.

Thanks.
